content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

```diff
@@ -24,8 +24,95 @@ further_reading:
 
 ## Automatic injection
 
-Given the many different ways to implement logging in PHP<span class="x x-first x-last">,</span> with some completely circumventing PHP's built-in error-logging API, the Datadog PHP tracing library cannot reliably inject trace and span <span class="x x-first x-last">IDs</span> into logs automatically.
-See the section below to learn how to connect your PHP Logs and traces manually.
+Starting in version `0.89.0`, the PHP tracer automatically injects trace correlation identifiers into application logs. To enable automatic injection, set the environment `DD_TRACE_LOGS_ENABLED` variable (INI setting `datadog.trace.logs_enabled`) to `true`.
```

Comment:
```suggestion
Starting in version `0.89.0`, the PHP tracer automatically injects trace correlation identifiers into application logs. To enable automatic injection, set the environment `DD_LOGS_INJECTION` variable (INI setting `datadog.trace.logs_enabled`) to `true`.
```

Reply:
:oops: I forgot to change the ini setting. Maybe you did

Reply:
Wait, it seems like I pushed but my changes weren't made

Reply:
Yep, I have changed the ini as well :)